### PR TITLE
Fixed HW-53999: Remove "+" from the amount in the List Transaction and Transaction Detail UI

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

File: UITests/Receipts/TransactionDetailTests.swift [hunk 1]
```diff
@@ -17,27 +17,27 @@ class TransactionDetailTests: BaseTests {
 
     // Credit Transaction
     func testReceiptDetail_verifyCreditTransaction() {
-        let expectedDateValue = "Fri, May 24, 2019, 6:16 PM"
+        let expectedDateValue = "Fri, May 24, 2019, 9:16 PM"
```
Comment (jsaini-hw) [inline UITests/Receipts/TransactionDetailTests.swift]: why changed the time?

File: UITests/Receipts/TransactionDetailTests.swift [hunk 2]
```diff
@@ -17,27 +17,27 @@ class TransactionDetailTests: BaseTests {
...
 
         // FEE Section
-        verifyFeeSection(amountVal: "+6.00 USD", feeVal: "0.00 USD", transactionVal: "6.00 USD")
+        verifyFeeSection(amountVal: "6.00 USD", feeVal: "0.00 USD", transactionVal: "6.00 USD")
     }
 
     // Debit Transaction
     func testReceiptDetail_verifyDebitTransaction() {
-        let expectedDateValue = "Sun, May 12, 2019, 6:16 PM" // Sun, May 12, 2019, 6:16 PM
+        let expectedDateValue = "Sun, May 12, 2019, 9:16 PM" // Sun, May 12, 2019, 6:16 PM
```
Comment (jsaini-hw) [inline UITests/Receipts/TransactionDetailTests.swift]: why changed the time? Update the comment as well

File: UITests/Receipts/TransactionDetailTests.swift [hunk 3]
```diff
@@ -136,7 +136,7 @@ class TransactionDetailTests: BaseTests {
     private func verifyDetailSectionOptional() {
         let transactionVal = "8OxXefx5"
         let receiptVal = "3051579"
-        let dateVal = "Fri, May 3, 2019, 5:08 PM" // Fri, May 3, 2019, 5:08 PM in test environment
+        let dateVal = "Fri, May 3, 2019, 8:08 PM" // Fri, May 3, 2019, 5:08 PM in test environment
```
Comment (jsaini-hw) [inline UITests/Receipts/TransactionDetailTests.swift]: same as above?

File: UITests/Receipts/TransactionDetailTests.swift (hunk 1 above)
Comment (oleksii-lubianyi-epam) [reply UITests/Receipts/TransactionDetailTests.swift]: Restored

File: UITests/Receipts/TransactionDetailTests.swift (hunk 2 above)
Comment (oleksii-lubianyi-epam) [reply UITests/Receipts/TransactionDetailTests.swift]: Restored

File: UITests/Receipts/TransactionDetailTests.swift (hunk 3 above)
Comment (oleksii-lubianyi-epam) [reply UITests/Receipts/TransactionDetailTests.swift]: Restored

**Pull request merged**